tock/utilization/unit.py

```diff
@@ -49,11 +49,10 @@ def _get_unit_billing_data(users, unit, recent_periods=None, fiscal_year=False):
         start, end, utilization = utilization_report(users, unit=unit)
 
     if utilization:
-        data = utilization.values('username', 'first_name', 'last_name', 'billable', 'target')
+        data = utilization.values('user_data', 'billable', 'target')
```

Comment:
the use of `values()` here was probably done to make this page quicker/more efficient, but it could complicate you getting the data you need on the page. I'm not sure about getting the whole `user_data` object in values() and if that will work like you want it to. This may help if there's a problem: https://docs.djangoproject.com/en/3.1/ref/models/querysets/#values

Reply:
oooh yes it grabs the ID of the object rather than the object itself, that documentation was extremely useful.

Reply:
Hmmm. We may need to look at the Utilization object... is it storing username there? Is that OK? Will we have a mismatch? Not sure.

Reply:
if the ID itself is useful, I'd suggest getting it explicity, just for clarity... `values('id', 'billable', 'target)`

If you prefer, you could say primary key instead: `values('pk', billable', target')

It'll be the same, so whichever feels more right to you

Reply:
Oh, sorry, I mean -- instead of returning all of `user_data` it returns the a key-value pair of `user_data: $USER_DATA_ID`